web-api/openapi/user_management.yaml

```diff
@@ -138,11 +145,13 @@ components:
           items:
             $ref: '#/components/schemas/UserInteractionData'
     UserInteractionError:
-      allOf:
-        - $ref: './schemas/error.yml#/BaseError'
-        - type: object
-          required: [kind]
-          properties:
-            kind:
-              type: string
-              enum: [InvalidUserId, InvalidDocumentId]
+      oneOf:
+        - $ref: './schemas/error.yml#/GenericError'
+        - allOf:
```

Comment:
Depending on the code generation we might need to switch the order of the `oneOf` around to go from most-precise to least-precise.

Reply:
@frankpepermans

Reply:
I removed this commit because it breaks the generation in all targets :(